modules/cnf-topology-aware-lifecycle-operator-create-policies.adoc

```diff
@@ -0,0 +1,133 @@
+// Module included in the following assemblies:
+// Epic CNF-2600 (CNF-2133) (4.10), Story TELCODOCS-285
+// * scalability_and_performance/cnf-topology-aware-lifecycle-operator.adoc
+
+:_content-type: PROCEDURE
+[id="talo-create-policies_{context}"]
+= Creating update policies for managed clusters
+
+You can create a `PolicyGenTemplate` CR with an {rh-rhacm} policy that updates the targeted clusters from the `ClusterVersion` object to the desired version, which then starts an update.
+
+.Prerequisites
+
+* Install the Topology Aware Lifecycle Operator.
+* Provision one or managed SNO clusters using ZTP.
+* Log in as a user with `cluster-admin` privileges.
+
+.Procedure
+
+. Create the `PolicyGenTemplate` CR in `inform` mode, then save the YAML in the `du-upgrade.yaml` file.
++
+[source,yaml]
+----
+apiVersion: ran.openshift.io/v1
+kind: PolicyGenTemplate
+metadata:
+ name: "du-upgrade"
+ namespace: "ztp-group-du-sno"
+spec:
+ bindingRules:
+   group-du-sno: "" <1>
+ mcp: "master"
+ remediationAction: inform
+ sourceFiles:
+   - fileName: ClusterVersion.yaml <2>
+     policyName: "platform-upgrade" <3>
+     metadata:
+       name: version
+     spec:
+       channel: "stable-4.9"
+       desiredUpdate:
+         version: "4.9.15" <4>
+----
```

Comment:
I agree. TALO works with ACM inform policies. In the example, we used PolicyGenTemplates (PGT) as a way to create ACM Policies (it is convenient in the ZTP workflow). However, an end-user may create directly ACM inform Policies in the hub cluster, either from a manifest or even from the ACM User Interface. TALO does not care, it just requires inform Policies in the hub cluster to start working on applying them to the managed clusters.

If we provide a PGT example we should briefly explain what is ZTP and link to the ZTP documentation. I think it makes more sense to just show an inform Policy we want to apply and comment that it can be created in multiple ways. WDYT?

Reply:
I added a prereq to the "Applying update policies to managed clusters"
> Create an RHACM policy in the Hub cluster.

Reply:
@amolnar-rh , I'm sorry, I don't really understand your comment. I think all this file should be restructured to explain that TALO works with inform policies created either by PGT (include link to PGT doc, if it exists) or by the user, as @alosadagrande has mentioned.